src/main.js

```diff
@@ -378,6 +393,8 @@ function handleAppWebContentsCreated(dc, contents) {
 }
 
 function initializeAfterAppReady() {
+  app.setAppUserModelId('Mattermost.Desktop'); // Use explicit AppUserModelID
```

Comment:
Just to double check, this ID is supposed to be changing from the previous one that referenced Squirrel, right?

Reply:
Yes, as part of investigating notifications for MSI, we decided to change to this. I removed all remaining reference to Squirrel as it wasn't all removed, so change this here as well.

Reply:
We discovered that this is apparently the convention to use for Windows for this property.